Exercise_6/ReversePolishNotationSpec.js

```diff
@@ -0,0 +1,95 @@
+const assert = require('assert');
+const chai = require('chai');
+const calculate = require('./ReversePolishNotation').calculate;
+
+describe('ReversePolishNotation', () => {
+
+    describe("Check single digit", () => {
+        it('Should return the same digit ', () => {
+            assert.equal(calculate('0'), 0);
+        })
+    })
+
+    describe("Single operations", () => {
+        it('Should return the sum', () => {
+            assert.equal(calculate('2 3 4 5 6 + + + +'), 20);
+        })
+        it('Should return the subtraction', () => {
+            assert.equal(calculate('4 2 - 3 4 - -'), 3);
+        })
+        it('Should return the multiplication', () => {
+            assert.equal(calculate('12 3 * 4 *'), 144);
+        })
+    })
+
+    describe("Check mixed operations ", () => {
+        it('+ - *', () => {
+            assert.equal(calculate('5 1 2 + 4 * 3 - +'), 14);
+        })
+        it('+ *', () => {
+            assert.equal(calculate('4 2 5 * + 1 3 2 * + +'), 21);
+        })
+        it('+ -', () => {
+            assert.equal(calculate('2 3 + 4 12 1 3 - - + +'), 23);
+        })
+        it('* -', () => {
+            assert.equal(calculate('5 6 2 1 - * *'), 30);
+        })
+    })
+
+    describe("Check extra operators", () => {
+        it("add / to operators", () => {
+            assert.equal(calculate('4 2 5 * + 1 3 2 * + /',
+                {
+                    '/': (firstValue, secondValue) => {
+                        return secondValue / firstValue
+                    }
+                }), 2)
+        })
+
+        it("add * to operators", () => {
+            assert.equal(calculate('4 2 %',
+                {
+                    '%': (firstValue, secondValue) => {
+                        return secondValue % firstValue
+                    }
+                }), 0)
+        })
+    })
+
+    describe("throw error on invalid argument", () => {
```

Comment:
It should be something like `'when invalid argument'`. And make single quotes for all it's.